doc/user_guide/profiling.rst

```diff
@@ -519,14 +555,14 @@ This is the code created for this example::
           CALL m2_proxy%halo_exchange(depth=1)
         END IF 
         CALL psy_data%PreEnd()
-        CALL psy_data_1%PreStart("multi_functions_multi_invokes_psy", "invoke_0:r2", &
+        CALL psy_data_1%PreStart("multi_functions_multi_invokes_psy", "invoke_0-r2", &
                                      0, 0)
         DO cell=1,mesh%get_last_halo_cell(1)
           CALL testkern_code(...)
         END DO 
         ...
         CALL psy_data_2%PreStart("multi_functions_multi_invokes_psy", &
-                          "invoke_0:testkern_code:r3", 0, 0)
+                          "invoke_0-testkern_code-r3", 0, 0)
```

Comment:
Again, no colons here at all now?

Reply:
That is (and was always) correct, PSyData takes two identifiers `(module_name, region_name)`. The change in this PS is only to replace the `:` in the **region** component.

How these two identifiers are presented to the user is then up to the wrapper library. Most of them use a `:` to make a single name (I did originally replace them as well, till I realised it's actually better to have the `:` in the profiling names, and I think you preferred that as well).